ci/pipeline

```diff
@@ -282,83 +364,5 @@ def release(requestVersion: String, gitSha: String, latest: Boolean = false): Un
-  *
-  * Also, we assert that rev contains new commits since the last release. If the proposal is to tag v1.5.1 with same
-  * revision as v1.5.0, then we fail, saying that there v1.5.0 points to the commit porposed for v1.5.1
-  *
-  * @param tagName The tag name / version we are going to tag. eg v1.5.1
-  * @param gitSha The commit sha / rev that will be tagged.
-  */
-def assertPriorVersionIsAncestor(tagName: String, gitSha: String): Unit = {
-  utils.priorPatchVersion(tagName).foreach { prior =>
-    val revs = %%("git", "rev-list", "--format=oneline", s"${gitSha}..${prior}").out.string
-    if(revs.trim.nonEmpty) {
-      println(s"Error! ${prior} is not a direct ancestor of proposed commit ${gitSha} for ${tagName}!")
-      sys.exit(1)
-    }
-
-    val commitsSinceLast = %%("git", "rev-list", "--format=oneline", s"${prior}..${gitSha}").out.string
-    if(commitsSinceLast.trim.isEmpty) {
-      println(s"Error! ${prior} points to proposed commit ${gitSha} for ${tagName}!")
-      sys.exit(1)
-    }
-  }
-}
-
-def withoutVersionPrefix(releaseVersion: String): String = {
-    if(releaseVersion.toLowerCase.startsWith("v"))
-      releaseVersion.substring(1)
-    else
-      releaseVersion
-}
-
-def uploadLinuxPackagesToRepos(tagName: String): Unit = {
-  val pkgserverUser = sys.env.getOrElse("PKG_SSH_USER", {
-    throw new IllegalStateException("PKG_SSH_USER environment variable must be set")
-  })
-  val pkgserverHost = sys.env.getOrElse("PKG_SSH_HOST", {
-    throw new IllegalStateException("PKG_SSH_HOST environment variable must be set")
-  })
-
-  // Note - the key is expected to be provided via an SSH agent
-  utils.printStageTitle(s"Uploading native packages")
-  %("rsync", "-avz",
-    (pwd / 'target / 'packages) + "/",
-    s"${pkgserverUser}@${pkgserverHost}:repo/incoming/marathon-${tagName}/")
-
-  val pkgType = if (tagName.toLowerCase contains "rc")
-    "-testing"
-  else
-    ""
-
-  val mappings = Seq(
-    "systemd" -> s"debian/jessie${pkgType}",
-    "systemd" -> s"ubuntu/yakkety${pkgType}",
-    "systemd" -> s"ubuntu/xenial${pkgType}",
-    "systemd" -> s"ubuntu/wily${pkgType}",
-    "systemd" -> s"ubuntu/vivid${pkgType}",
-    "upstart" -> s"ubuntu/trusty${pkgType}",
-    "upstart" -> s"ubuntu/precise${pkgType}",
-    "systemv" -> s"el${pkgType}/6",
-    "systemd" -> s"el${pkgType}/7")
-
-  val copyCommands = mappings.map { case (packageType, path) =>
-    s"cp $$HOME/repo/incoming/marathon-${tagName}/${packageType}-marathon* " +
-    s"$$HOME/repo/incoming/${path}/"
-  }.mkString(";")
-
-  utils.printStageTitle("Distributing packages to distros")
-  %("ssh", s"${pkgserverUser}@${pkgserverHost}", "bash",
-    "-e", "-x", "-c",
-    utils.escapeCmdArg(List(
-      copyCommands,
-      s"rm -rf $$HOME/repo/incoming/marathon-${tagName}").mkString("\n")))
-
-  utils.printStageTitle("All done")
+  // TODO: Publish native packages to stable.
```

Comment:
Is't this `TODO` already DONE with `uploadLinuxPackagesToRepos`?

Reply:
This target is not called.